image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -72,6 +72,8 @@ sensor:
   exposeMonitoring: null # bool
   nodeSelector: null # string | dict
   tolerations: null # [dict]
+  localImageScanning:
```

Comment:
Nit: I don't know exactly where, but it would be nice to document what does this mean exactly, in particular as opposed to `scanner.disable`. I understand them as:
- localImageScanning.enabled == "some scanner is available in the same namespace as sensor"
- scanner.disable == ! "this chart should launch a scanner"

For me having that in advance helps understanding the code much faster, and I don't think that meaning is obvious from the current naming

Reply:
This enables the local image scanning feature in Sensor. It is disabled if Sensor should not use the feature. 

scanner.disable determines whether Scanner is deployed or not.

Reply:
Added docs.